.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
These seem to be different from the linter options proposed by Matthew. For consistency sake, it may be worth standardizing between the front end and the servers. Are there any requirements here that are specific to React Native?

Reply:
This is just a basic template with the main goal of just getting react native setup. Matthew is free to update or change any eslint styles to the ones discussed.